src/services/message.service.js

```diff
@@ -0,0 +1,31 @@
+const { ApiError } = require('../exceptions/apiError');
+const { Message } = require('../models/Message');
+const { join } = require('./room.service');
+
+const createMessage = async (author, text, roomId) => {
+  if (!author) {
+    throw ApiError.unauthorized();
+  }
+
+  if (!text) {
+    throw ApiError.badRequest('Message is required');
+  }
+
+  const existRoom = await join(roomId);
+
+  if (!existRoom) {
+    throw ApiError.notFound();
+  }
+
+  const message = await Message.create({ author, text, roomId });
+
+  return message;
+};
+
+const getMessages = async (id) => {
+  const messages = await Message.findAll({ where: { id } });
```

Comment:
The `getMessages` function retrieves messages by their ID. Typically, you might want to retrieve messages by `roomId` to get all messages for a specific room. Consider changing the query to `where: { roomId: id }` if that aligns with your application's requirements.